docs/guide/java_object_graph_guide.md

```diff
@@ -161,6 +161,28 @@ If a number are `long` type, it can't be represented by smaller bytes mostly, th
 not worthy compared to performance cost. Maybe you should try to disable long compression if you find it didn't bring much
 space savings.
 
```

Comment:
Can we move this as first section of advanced?

Reply:
Done.